metrics/ci_integrations/lib/source/jenkins/adapter/jenkins_source_client_adapter.dart

```diff
@@ -231,9 +235,16 @@ class JenkinsSourceClientAdapter with LoggerMixin implements SourceClient {
     return _processInteraction(interaction);
   }
 
-  /// Maps the [result] of a [JenkinsBuild] to the [BuildStatus].
-  BuildStatus _mapJenkinsBuildResult(JenkinsBuildResult result) {
-    switch (result) {
+  /// Maps the given [JenkinsBuild.building] and [JenkinsBuild.result] of the
+  /// given [build] to a [BuildStatus].
+  ///
+  /// Returns the [BuildStatus.inProgress] if the given [JenkinsBuild.building]
+  /// is `true`. Otherwise, maps the given [JenkinsBuild.result] to a
+  /// corresponding [BuildStatus].
```

Comment:
```suggestion
  /// Maps the [JenkinsBuild] result to a corresponding [BuildStatus] using 
  /// the given [building] and [result] parameters.
  ///
  /// Returns the [BuildStatus.inProgress] if the given [building]
  /// is `true`. Otherwise, maps the given [result] to the
  /// corresponding [BuildStatus].
```